test/mocks/in-memory-mongo-store.ts

```diff
@@ -1,15 +1,14 @@
-import { MongoMemoryServer } from 'mongodb-memory-server';
 import { connect, Mongoose as M } from 'mongoose';
 import { Mongoose } from '@storage';
+import config from 'config';
 
 export class InMemoryMongoStore extends Mongoose.MongoStore {
   constructor() {
     super();
   }
 
   public async connect(): Promise<M> {
-    const mongod = new MongoMemoryServer();
-    const uri = await mongod.getConnectionString();
+    const uri: string = config.get('mongoDb.uri');
```

Comment:
Use `mongod.getUri()`. We should use MongoMemoryServer instead of our database for testing.

Reply:
Didn't got this.

Reply:
updated